.io-config.json

```diff
@@ -0,0 +1 @@
+{"app_id":"66d1ea4e","api_key":"b29c0fee3007e42d1d16a3f547c469865283eff9089fb0d8"}
```

Comment:
while these may be necessary for you to get everything to run, please don't include them as part of the pull request. We have a particular `app_id` and `api_key` for our project on the ionic platform and overwriting it means that ionic deploy won't work any more.